Ulabox/Sniffs/Commenting/FunctionCommentSniff.php

```diff
@@ -146,8 +148,77 @@ protected function processReturn(PHP_CodeSniffer_File $phpcsFile, $stackPtr, $co
                     $phpcsFile->addError($error, $tokens[$commentStart]['comment_closer'], 'MissingReturn');
                 }
             }
-        }//end if
-    }//end processReturn()
+        }
+    }
+
+    /**
+     * Returns a valid variable type for param/var tag.
+     *
+     * If type is not one of the standard type, it must be a custom type.
+     * Returns the correct type name suggestion if type name is invalid.
+     *
+     * @param string $varType The variable type to process.
+     *
+     * @return string
+     */
+    private function suggestType($varType)
+    {
```

Comment:
I seems like the order of *public*, *protected* and *private* of my IDE's autoformatter may not be the correct. Let me check it! Thanks!